buildbot-host/aptly/fpm/package.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+#
+# Create "publish-incoming-debian-snapshots" Debian package
+#
+set -e
+
+SRC=$(dirname -- "$( readlink -f -- "$0"; )";)/..
+FILES_FOUND=yes
+
+if [ "x${SRC}" = "x" ]; then
+  echo "ERROR: unable to determine source directory!"
+  exit 1
+fi
+
+for FILE in \
+    "${SRC}/publish-debian-snapshot.sh" \
+    "${SRC}/publish-incoming-debian-snapshots.sh" \
+    "${SRC}/publish-incoming-debian-snapshots.service"
+do
+    if ! [ -r "${FILE}" ]; then
+        echo "ERROR: ${FILE} missing, cannot build!"
+	FILES_FOUND=no
+    fi
+done
+
+if [ "${FILES_FOUND}" = "no" ]; then
```

Comment:
This is by design. The idea is to get an error for every file that is missing instead of always failing on the first. That said, the code is probably a bit too fancy: all the files it needs are already available in the Git repository anyways and unlikely to be missing.